src/widgets/Logo.tsx

```diff
@@ -35,7 +35,7 @@ export default class Logo extends React.Component<Props> {
       <Cell data-testid="widget" area={area} center style={{ backgroundColor: "#292A29" }}>
         <WidgetContainer>
           <WidgetTitle>CDS Dashboard</WidgetTitle>
-          <CdsLogo />
+          <CdsLogoEn />
```

Comment:
This component isn't currently used so I haven't implement the language switching yet.